config/production.json

```diff
@@ -1,7 +1,4 @@
 {
-  "services": {
-    "declarationsPath": "../declarations/declarations"
```

Comment:
Why is this changed? I'm worried of consequences on production instances.

Reply:
It's obsolete and overridden by instances: https://github.com/OpenTermsArchive/contrib-declarations/pull/748/files#diff-07e0c2ac42cdc22639d12b2282f02825cda2dcf94c741cad1f57706e93b62d2fR5